clash-lib/tests/Test/Clash/Rewrite.hs

```diff
@@ -60,16 +60,28 @@ lookupTM u tm = case HashMap.lookup u tm of
 
 instance Default RewriteEnv where
   def = RewriteEnv
-    { _debugOpts=debugSilent
-    , _aggressiveXOpt=False
+    { _clashEnv = ClashEnv
+        { envOpts = defClashOpts
+            { opt_debug = debugSilent
+            , opt_aggressiveXOpt = False
+            , opt_inlineLimit = 20
+            , opt_inlineFunctionLimit = 15
+            , opt_inlineConstantLimit = 0
+            , opt_inlineWFCacheLimit = 10
+            , opt_specLimit = 20
+            , opt_evaluatorFuelLimit = 10
+            , opt_newInlineStrat = True
+            , opt_ultra = False
+            }
```

Comment:
Why aren't these part of `defClashOpts`?

Reply:
Not sure, I didn't double check _how_ they differed from `defClashOpts`. I wouldn't expect anything in this test to need limits raising...

Reply:
The only meaningful change was `opt_debug` so I removed the others